acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -179,6 +184,49 @@ public void createAzureKeyYamlFileAt(
     }
   }
 
+  public void createYubiHsmYamlFileAt(
+      final Path metadataFilePath,
+      final Path destinationYubiShellSimulator,
+      final KeyType keyType) {
+    final int opaqueObjId = keyType == KeyType.BLS ? 1 : 2;
+
+    final Map<String, Serializable> yaml =
+        Map.of(
+            "type",
+            "yubihsm2",
+            "yubiShellBinaryPath",
+            destinationYubiShellSimulator.toString(),
```

Comment:
not sure I love the executable being in the config file - this means you can't reuse this file-shape if we changed to using the yubihsm as an embedded library - maybe this should come in via Env Var (with default value?)

Reply:
We can assume that the binary `yubihsm-shell` is available on global PATH, and can be overridden from environment variable `WEB3SIGNER_YUBISHELL_PATH`